src/views/ApartmentApp/components/StudentApplication/components/ApplicantList/components/ApplicantListItem/index.js

```diff
@@ -121,7 +102,7 @@ const ApplicantListItem = ({
               <IconButton
                 aria-label={isApplicationEditor ? 'current-editor' : 'set-new-editor'}
                 disabled={isApplicationEditor || disabled}
-                onClick={handleChangeEditor}
+                onClick={() => profile && onChangeEditor?.(profile)}
```

Comment:
So are there cases where the `onChangeEditor` prop is `null`? That seems like an odd design choice.

Reply:
This is done to allow you to omit the onChangeEditor from the props entirely and still have the component display correctly. In those cases, it is specifically handled as `undefined`, so I did some research and learned that you can use the `?.` for functions, and I did a bunch of experimentation in CodePen to test the behavior in a bunch of possible situations

This "feature" is mostly used for when the user is not the editor of an application, so they are allowed to view all the stuff but all the interactive things are disabled. Thus there is no need to pass the functions, and having them undefined helps prevent things from accidentally triggering when they are not supposed to.